src/numberdrome.js

```diff
@@ -0,0 +1,72 @@
+class Numberdrome {
+    constructor(array) {
+        this.array = array;
+    }
+
+    addNumber(n) {
+        return this.array.push(n);
+    }
+
+    removeNumber(n) {
+        debugger;
+        return this.array.splice(this.array.indexOf(n), 1);
+    }
+
+    sum() {
+        let sum = 0;
+        for (let number of this.array) {
+            sum += number;
+        }
+        return sum;
+    }
+
+    product() {
+        let product = 1;
+        for (let number of this.array) {
+            product *= number;
+        }
+        return product;
+    }
+
+    min() {
+        if (this.array.length === 0) {
```

Comment:
перевірку на 0 роблять так
```suggestion
        if (!this.array.length) {
```